.circleci/config.yml

```diff
@@ -77,29 +32,29 @@ jobs:
       - checkout
       - restore_cache:
           keys: python3.8
-      - requirements:
-          python_version: "3.8"
+      - requirements
       - run:
-          name: Run the tests using Python 3.8
-          command: './venv/bin/tox -e "py{38}-django{22}"'
+          name: Run tests using Python 3.8
+          command: 'tox -e "py{38}-django{22}"'
       - save_cache:
           key: python3.8
           paths:
             - .tox
             - /home/circleci/.cache/pip
+
   pii_check:
     docker:
-      - image: circleci/python:3.5
+      - image: circleci/python:3.8
     steps:
       - checkout
       - restore_cache:
-          keys: python3.5
+          keys: python3.8
       - requirements
       - run:
-          name: Run the PII checks
-          command: "./venv/bin/tox -e pii_check"
+          name: Run PII checks
+          command: 'tox -e pii_check'
```

Comment:
Nit: I believe these quotes are redundant.

Reply:
Yes, they were. I've removed them in the latest commit.